tool.gpt

```diff
@@ -0,0 +1,37 @@
+Name: List Datasets
+Description: Lists all available datasets
+
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool listDatasets
+
+---
+Name: List Elements
+Description: Lists metadata for all elements in a dataset
+Param: datasetID: the ID of the dataset
```

Comment:
nit: I believe using `snake_case` for parameter names will lead to GPTScript injecting more legible environment variables; e.g. `dataset_id` -> `DATASET_ID`.

Reply:
yeah, I originally had it as snake case but changed it so that the strings in the json struct tags would be nicer (without having to write needless code to convert it) over in the SDKServer code. So I'm just gonna keep it like this here.